RELEASING.md

```diff
@@ -2,7 +2,31 @@
 
 This document explains how to publish all OT modules at version x.y.z. Ensure that you’re following semver when choosing a version number.
 
-Release Process:
+## Auto-generate a Release PR
+
+Navigate to the [Create a Release PR Worflow](https://github.com/open-telemetry/opentelemetry-js-contrib/actions/workflows/release-pr.yml). Click the "Run workflow" dropdown, enter the semver-compliant version to release, then click "Run workflow". It will generate a release pull request from a branch named like `release/x.y.z`.
+
+## Review Release PR
+
+If necessary, update sample code or documentation on the `release/x.y.z` branch to be included in the release. Another maintainer should approve the PR, specifically validating the automatic updates and verifying the tests are passing.
+
+NOTE: If any major changes are merged while the release PR is open, the CHANGELOG on the `release/x.y.z` branch must be manually updated to reflect the change.
+
+## Merge Release PR
+
+After approval, merge the release PR. This will automatically publish all packages. Verify:
+
+* The [publish workflow](https://github.com/open-telemetry/opentelemetry-js-contrib/actions/workflows/publish.yml) runs successfully
+* The new version is available in NPM, e.g. for the [Jaeger Propagator](https://www.npmjs.com/package/@opentelemetry/propagator-jaeger)
```

Comment:
could you use another example ? The jeager propagator is moving out to core repo (https://github.com/open-telemetry/opentelemetry-js-contrib/pull/351)

Reply:
Bad random choice :) switched to Express instrumentation